src/t_zset.c

```diff
@@ -2558,7 +2558,8 @@ dictType setAccumulatorDictType = {
  *
  * 'op' SET_OP_INTER, SET_OP_UNION or SET_OP_DIFF.
  */
-void zunionInterDiffGenericCommand(client *c, robj *dstkey, int numkeysIndex, int op) {
+void zunionInterDiffGenericCommand(client *c, robj *dstkey, int numkeysIndex, int op,
+                                   int cardinality_only) {
```

Comment:
Doc comment needs to be updated and reflect that the new arg is only applicable to intersection.